common/sgx/quote.c

```diff
@@ -529,6 +545,208 @@ oe_result_t oe_verify_quote_with_sgx_endorsements(
     }
 
     oe_datetime_log("Validation datetime: ", &validation_time);
+
+    // Convert validation time to time_t
+    OE_CHECK(oe_datetime_to_time_t(&validation_time, &expiration_check_date));
+
+    // Try to call SGX DCAP QVL to verify quote first
+#ifdef OE_BUILD_ENCLAVE
```

Comment:
Style suggestion: long ifdefs are harder to review and maintain.  Can you factor the code for each of the two cases (ifdef/else) out into a separate function and call them here?

Reply:
Make sense. But I prefer to submit another PR to enhance the code style.